mps-model-adapters-plugin/src/test/kotlin/org/modelix/model/mpsadapters/ChangePropertyTest.kt

```diff
@@ -28,10 +29,11 @@ class ChangePropertyTest : MpsAdaptersTestBase("SimpleProject") {
 
         val repositoryNode: INode = MPSRepositoryAsNode(mpsProject.repository)
 
-        writeActionOnEdt {
+        runCommandOnEDT {
             val module = repositoryNode.getChildren(BuiltinLanguages.MPSRepositoryConcepts.Repository.modules)
                 .single { it.getPropertyValue(BuiltinLanguages.jetbrains_mps_lang_core.INamedConcept.name) == "Solution1" }
-            val model = module.getChildren(BuiltinLanguages.MPSRepositoryConcepts.Module.models).single()
+            val model = module.getChildren(BuiltinLanguages.MPSRepositoryConcepts.Module.models)
+                .single { (it as MPSModelAsNode).model !is GenericDescriptorModelProvider.DescriptorModel }
```

Comment:
@mhuster23  Is it expected behavior that our adapter makes `GenericDescriptorModelProvider.DescriptorModel`s available?

Reply:
I created an issue for that. 
See [MODELIX-918](https://issues.modelix.org/issue/MODELIX-918)